packages/app/src/composables/useToken.ts

```diff
@@ -20,10 +22,13 @@ export const retrieveToken = useMemoize(
     },
   }
 );
-
+export const retrieveERC20NativeToken = useMemoize(
```

Comment:
What is `retrieveERC20NativeToken` for? Also `api/token/nativeERC20Info` API endpoint doesn't exist.

Reply:
This was an old endpoint for a first iteration of this PR, removed.